src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderTest.cs

```diff
@@ -790,6 +790,95 @@ static char[] Resize(char[] buffer)
+
+            using (var cn = new Microsoft.Data.SqlClient.SqlConnection(DataTestUtility.TCPConnectionString))
+            {
+                await cn.OpenAsync();
+
+                string tableName = DataTestUtility.GenerateObjectName();
+
+                try
+                {
+                    using (var cmd = cn.CreateCommand())
+                    {
+                        cmd.CommandText = $"""
+                   if exists (select *
```

Comment:
I think your indents got messed up here

Reply:
it's using """'s so the indents are handled by the compiler. Not a feature I've used before but it's part of the original reproduction and it works so I left it alone.